### PR TITLE
fix: unable to unselect boolean flag

### PR DESCRIPTION
**Pull request by DmytroHryshyn**

### CODE REVIEW

**Comment by DmytroHryshyn** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #853** <a href="https://app.graphite.dev/github/pr/intuita-inc/intuita-vscode-extension/853?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/intuita-inc/intuita-vscode-extension/853?utm_source=stack-comment).

**Review by hbjORbj — APPROVED** [review]

File: intuita-webview/src/codemodList/CodemodArguments/index.tsx [hunk 1]
```diff
@@ -45,18 +45,14 @@ const CodemodArguments = ({
...
-				.value;
-
-			vscode.postMessage({
-				kind: 'webview.global.setCodemodArgument',
-				hashDigest,
-				name: fieldName,
-				value,
-			});
-		};
+	const onChangeFormField = (fieldName: string) => (value: string) => {
```
Comment (grzpab) [inline intuita-webview/src/codemodList/CodemodArguments/index.tsx:48]: do you think we should type `fieldName` as a type literal to ensure proper names?

File: intuita-webview/src/codemodList/CodemodArguments/FormField/index.tsx [hunk 2]
```diff
@@ -18,7 +18,8 @@ const FormField = (props: Props) => {
 			<VSCodeTextField
 				placeholder={name}
 				value={String(value)}
-				onInput={onChange}
+				// @ts-expect-error value exists on target
```
Comment (grzpab) [inline intuita-webview/src/codemodList/CodemodArguments/FormField/index.tsx:21]: I would not use `@ts-expect-error` with complex ASTs, this directive disables the checks for the entire line (it's not targeting a specific issue).
I'd rather check the event for the target value and execute `onChange` only if the latter exists.

File: intuita-webview/src/codemodList/CodemodArguments/index.tsx (hunk 1 above)
Comment (DmytroHryshyn) [reply intuita-webview/src/codemodList/CodemodArguments/index.tsx:48]: we already have type literal for `kind`, `fieldName` is just a user-defined argument name

**Pull request merged**